nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/impl/AbstractDocumentModelWriter.java

```diff
@@ -382,7 +383,18 @@ private static Object getElementData(ExportedDocument xdoc, Element element, Typ
                 }
                 if (blob == null) { // maybe the blob is embedded in Base64
                     // encoded data
-                    byte[] bytes = Base64.decodeBase64(content);
+                    byte[] bytes;
+                    try {
+                        bytes = Base64.decodeBase64(content);
+                    } catch (IllegalArgumentException e) {
+                        // example invalid base64: fd7b9e4.blob
+                        if (log.isDebugEnabled()) {
+                            log.warn("Invalid blob base64 in document: " + xdoc.getId() + ": " + StringUtils.abbreviate(content, 50));
```

Comment:
What's the point of logging the beginning of the content?

Reply:
It allows for better debugging, to find out where the invalid content comes from.